packages/starknet-snap/index.html

```diff
@@ -507,6 +521,16 @@ <h1>Hello, Snaps!</h1>
       await callSnap('starkNet_signMessage', { typedDataMessage, signerAddress });
     }
 
+    // here we call the snap's "starkNet_signTransaction" method
+    async function signTransaction(e) {
+      e.preventDefault(); // to prevent default form behavior
+
+      const transaction = document.getElementById('transactions').value;
+      const transactionsDetail = document.getElementById('transactionsDetail').value;
+
+      await callSnap('starkNet_signTransaction', { transaction, transactionsDetail });
```

Comment:
the transaction and transactionsDetail may not able to passin to callSnap due to html form

Reply:
what do you suggest here ?